walkthroughs/howto-tls-file-provided/README.md

```diff
@@ -279,12 +275,6 @@ Now when you call the service, you will see both `white` and `green` again.
 curl "${COLORAPP_ENDPOINT}/color"
 ```
 
-Tip: You can use this command to clear the color teller history
-
-```bash
```

Comment:
We can still retain this.

Reply:
Without the color teller gateway, we actually only return the color. So `curl http://endpoint....` will return `white` or `green` and not the `{"green": .5, "white": .5}` that we saw before

Reply:
I see. I was under the assumption that `/clear` is provided by the colorteller image.